### ISSUE TITLE
{"Object reference not set to an instance of an object."}

### ISSUE THREAD

**Issue by lmcgs** [Buzzilla/webhose-dotnet#3]
When i try get a response like that:

`WebhoseRequest clientRequest = new WebhoseRequest(APIKEY);
WebhoseResponse response = clientRequest.getResponse("minecraft");`

**Comment by IdanHaim** [comment]
Your getting this error because the clientRequest is null.
Please check your APIKEY that he is the right one and then try again.

**Comment by lmcgs** [comment]
No. Not about APIKEY. I found and handle it. There was no null control in WebhosePost.cs for post["persons"] and post["locations"] and post["organizations"]. I added null controls. And now everything working fine.

**Comment by aherrera818** [comment]
Yeah same here , I just added some null checks , and its working fine.

**Comment by IdanHaim** [comment]
https://github.com/Buzzilla/webhose-dotnet/pull/5
